src/Bundle/DependencyInjection/SyliusGridExtension.php

```diff
@@ -57,6 +59,10 @@ public function load(array $configs, ContainerBuilder $container): void
         $container->registerForAutoconfiguration(GridInterface::class)
             ->addTag('sylius.grid')
         ;
+
+        if (!\class_exists(AbstractMaker::class)) {
```

Comment:
Can we somehow write test case for it? I'm not sure. The best idea I have right now is to duplicate test app without maker bundle... Did you test it manually?

Reply:
I didn't test it manually, but I have an idea....

Reply:
Voilà!
https://github.com/Sylius/SyliusGridBundle/pull/206/commits/b76ec9fcfab96d32fde3d752b08851231124e3b8

Reply:
done... it wasn't as simple as I thought 🙄